lib/mapper.js

```diff
@@ -18,10 +19,25 @@ var Track = require('segmentio-facade').Track;
  */
 
 exports.identify = function(identify){
-  return {
-    token: this.settings.apiKey,
-    properties: traits(identify)
+  var opts = identify.options(this.name);
+  var msg = {
+    peopleData: {
+      token: this.settings.apiKey,
+      properties: traits(identify)
+    }
   };
+
+  // email, listId, and api_key all required
+  if (opts.listId && identify.email() && this.settings.privateKey) {
+    msg.listId = opts.listId;
+    msg.listData = {
+      email: identify.email(),
+      api_key: this.settings.privateKey,
+      confirm_optin: opts.confirmOptin != undefined ? opts.confirmOptin : this.settings.confirmOptin // default setting is true, can be overriden per event via options
```

Comment:
i think we have prior art here in our mailchimp integration and would like to handle it the same way we do for them